bundle-size/github-utils.js

```diff
@@ -40,7 +40,7 @@ function getBuildArtifactsFileParams_(filename) {
 }
 
 /**
- * Utils for GitHub actions.
+ * Utils for GitHub actions that are performed by the user based token.
```

Comment:
Which account does this token  `process.env.ACCESS_TOKEN` belong to?

Reply:
@ampprojectbot